projects/addon-doc/components/example/example.template.html

```diff
@@ -1,26 +1,28 @@
-<h3
-    *ngIf="heading"
-    class="t-title"
->
-    <span
-        *polymorpheusOutlet="heading as text"
-        [textContent]="text"
-    ></span>
+<div class="t-title-block">
+    <h3
+        *ngIf="heading"
+        class="t-title"
+    >
+        <span
+            *polymorpheusOutlet="heading as text"
+            [textContent]="text"
+        ></span>
+    </h3>
     <a
         *ngIf="id"
-        appearance="icon"
         routerLink="."
-        size="xs"
-        tuiIconButton
+        routerLinkActive="t-link-active"
+        tuiLink
         type="button"
-        class="t-link-icon"
+        class="t-link"
+        [attr.title]="copy$ | async"
         [fragment]="id"
-        [iconStart]="icons.link"
+        [routerLinkActiveOptions]="{matrixParams: 'exact', queryParams: 'exact', paths: 'exact', fragment: 'exact'}"
```

Comment:
I think it can be just `{ exact: true }`?

![image](https://github.com/user-attachments/assets/d21577b4-9822-40ac-83e9-8ee8b52db3be)

Reply:
<img width="507" alt="image" src="https://github.com/user-attachments/assets/4dbcafd1-e4c5-4ed3-b3db-63510cb66806">

```
[routerLinkActiveOptions]="{matrixParams: 'exact', queryParams: 'exact', paths: 'exact', fragment: 'exact'}"
```

<img width="538" alt="image" src="https://github.com/user-attachments/assets/93d2939d-2651-4e66-9cce-0a18772fa22b">